### ISSUE TITLE
automate the addition of default cf attributes?

### ISSUE THREAD

**Issue by jerabaul29** [OpenDrift/trajan#146]
Should we automate the addition of default cf attributes when these are missing? Or should this be something the user has to call in manually? I.e. the addition of the attributes as done by:

```
ds = ds.traj.assign_cf_attrs()
```

Could look for the existence of the requested attributes in the class `__init__` when a `Traj` object is created for example, and call this automatically if the attributes are not there? Any thoughts about this @gauteh @knutfrode  ? :)

**Comment by knutfrode** [comment]
Maybe this could be added by default if we make a `ds.traj.to_netcdf()` (but still with possibility to skip)

**Comment by gauteh** [comment]
I don't think we should modify the dataset in __init__ without any action by the user. In theory it needs to be updated in a lot of cases, seltime, gridtime, isel(trajectory=0) (which we don't control). Maybe in to_netcdf is a good place. I was also wondering if this should be `normalize()` method, which also fixes other inconsistencies (like missing role attributes etc).
